*** main.c.org	2014-03-01 09:20:40.916004000 +0900
--- main.c	2014-02-26 11:11:42.056536400 +0900
*************** void ProcessIO(void)
*** 1184,1189 ****
--- 1184,1192 ----
              	{	/* 受信待ちモード設定 */
  	            	uc_read_code_type = CODE_TYPE_EXTENSION;
  	            	uc_receive_wait_mode = RECEIVE_WAIT_MODE_WAIT;
+ #ifdef __FOR_ANDROID
+                     send_read_data_flag = 0;
+ #endif
  	            }
  	            else
  	            {	/* 受信待ちモード解除 */
